src/victory-theme/grayscale.js

```diff
@@ -0,0 +1,134 @@
+const grayscaleBlack = "#252525";
+const fontStack = "Poppins, 'Century Gothic', 'Trebuchet MS', sans-serif";
+const letterSpace = 0.2;
+const labelFontSize = 10;
+const dataTickLabelPadding = 5;
+const strokeCap = "round";
+
+export default {
+  line: {
+    data: {
+      stroke: grayscaleBlack,
+      strokeWidth: 2,
+      fill: "none",
+      opacity: 1
+    },
+    labels: {
```

Comment:
A lot of stuff is repeated. Could you do something like `const baseLabelStyle = {...}` and then `assign` the differences?